### PR TITLE
Improve some command formatting

### PR DESCRIPTION
**Pull request by AndrewFerr**
- Ensure args included in monospace formatting
- Show required args as `<arg>`
- Show optional args as `[arg]`
- Don't print undefined username with Jira `whoami`
- Make `feed remove`'s wording consistent with the feed add command

### CODE REVIEW

File: changelog.d/504.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Improve formatting of help commands and Jira's `whoami` command.
```
Comment (Half-Shot) [inline changelog.d/504.feature:1]: Arguably a `misc` or a `bugfix`?

File: src/BotCommands.ts [hunk 2]
```diff
@@ -49,10 +49,15 @@ export function compileBotCommands(...prototypes: Record<string, BotCommandFunct
...
+                const requiredArgs = b.requiredArgs?.map((arg: string) =>  `<${arg}>`).join(" ") || "";
                 const optionalArgs = b.optionalArgs?.map((arg: string) =>  `[${arg}]`).join(" ") || "";
+                const cmdStr =
+                    ` - \`££PREFIX££${b.prefix}` +
+                    (requiredArgs ? ` ${requiredArgs}` : "") +
+                    (optionalArgs ? ` ${optionalArgs}` : "") +
+                    `\` - ${b.help}`;
                 cmdStrs[category] = cmdStrs[category] || []
-                cmdStrs[category].push(` - \`££PREFIX££${b.prefix}\` ${requiredArgs} ${optionalArgs} - ${b.help}`);
+                cmdStrs[category].push(cmdStr);
```
Comment (Half-Shot) [inline src/BotCommands.ts:60]: Kinda surprised we don't have tests for this code. Oh well

**Review by Half-Shot — APPROVED** [review]

File: changelog.d/504.feature (hunk 1 above)
Comment (AndrewFerr) [reply changelog.d/504.feature]: Opted for `bugfix`

File: src/BotCommands.ts (hunk 2 above)
Comment (AndrewFerr) [reply src/BotCommands.ts:60]: Yeah, it would be nice, but at least it's extremely visible :smiley:

**Pull request merged**